R/render_graph.R

```diff
@@ -167,8 +167,7 @@ render_graph <- function(graph,
         "fillcolor" %in% colnames(graph$nodes_df)) {
 
       graph$nodes_df$fontcolor <-
-        graph$nodes_df$fillcolor %>%
-        dplyr::as_data_frame() %>%
+        tibble::tibble(value = graph$nodes_df$fillcolor) %>%
```

Comment:
This modification seems like a neutral-to-good idea, regardless.

Reply:
I think a solution that works with 1.4.2 is better, since the sequencing of the releases becomes simpler.